src/components/Charts/ChartFutureHospitalization.tsx

```diff
@@ -1,236 +0,0 @@
-import React, { useContext } from 'react';
-import { ThemeContext } from 'styled-components';
-import { last, isDate } from 'lodash';
-import { extent as d3extent, max as d3max } from 'd3-array';
-import { LinePath } from '@vx/shape';
-import { ParentSize } from '@vx/responsive';
-import { scaleLinear, scaleTime } from '@vx/scale';
-import { COLORS } from 'common';
-import { Projections } from 'common/models/Projections';
-import { assert, formatUtcDate, formatInteger } from 'common/utils';
-import { AxisBottom } from './Axis';
-import BoxedAnnotation from './BoxedAnnotation';
-import ChartContainer from './ChartContainer';
-import RectClipGroup from './RectClipGroup';
-import Tooltip from './Tooltip';
-import * as TooltipStyle from './Tooltip.style';
-import { LegendMarker, LegendLine } from './Legend';
```

Comment:
We can delete the [`Legend.tsx`](https://github.com/covid-projections/covid-projections/blob/d7b93f5fc5640d228d03ad9a5e8af7805a440c60/src/components/Charts/Legend.tsx) file as well. We can clean up a bit more in a follow-up PR as well (I imagine that some of the styles are not needed anymore)